### PR TITLE
22.09

### PR DESCRIPTION
**Pull request by mtawfik-95**

### CODE REVIEW

File: docs/source/usage.rst [hunk 1]
```diff
@@ -33,6 +33,32 @@ For example:
...
+uses `yield`-based semantics to both allow you to retrieve a large number of predictions while
+also maintaining low request latency and response size.
+
+.. code-block:: python
+
+   import datetime
+   from konan_sdk.sdk import KonanSDK
+
+   sdk = KonanSDK(verbose=False) # Initialize the SDK.
+   user = sdk.login("<email>", "<password>") # Login user your valid konan credentials
```
Comment (NourhanKhaled) [inline docs/source/usage.rst:49]: Can you replace this with the api key login or add a separate section for login with api key?

File: docs/source/usage.rst [hunk 2]
```diff
@@ -33,6 +33,32 @@ For example:
...
+
+   predictions_generator = sdk.get_predictions(
+      deployment_uuid="<deployment-uuid>",
+      start_time=datetime.datetime(year=2022, month=9, day=1),
+      end_time=datetime.datetime(year=2022, month=10, day=1),
+   )
+
+   for predictions in predictions_generator:
+      print(len(predictions))
+      print(predictions[0].uuid, predictions[0].features)
```
Comment (NourhanKhaled) [inline docs/source/usage.rst:59]: An average user might be confused with the generator syntax, maybe rename `predictions` to `predictions_page` for clarity or add a small comment explaining that the generator iterates over pages and each page you can iterate over the predictions themselves.

File: docs/source/usage.rst (hunk 1 above)
Comment (mtawfik-95) [reply docs/source/usage.rst]: Done

File: docs/source/usage.rst (hunk 2 above)
Comment (mtawfik-95) [reply docs/source/usage.rst:63]: Done

**Review by NourhanKhaled — APPROVED** [review]

**Pull request merged**